test/md2test.c

```diff
@@ -10,7 +10,6 @@
 #include <string.h>
 
 #include "../e_os.h"
```

Comment:
I'm much more inclined to do it the other way around, actually...  Not sure if we have a team concensus for this.